directives/questionnaire.py

```diff
@@ -313,6 +315,9 @@ def split_second_last(empty_lines):
             if key.startswith(u'*'):
                 correct = True
                 key = key[1:]
+            if key.startswith(u'?'):
```

Comment:
I suppose the `key` should never have both `*` and `?`, so if statement could use `elif`.